content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,135 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
+                        [-y <field>] [--no-header] [--title <text>]
+                        [--x-label <text>] [--y-label <text>]
+                        [--unset [<property> [<property> ...]]]
+                        target
+
+positional arguments:
+  target                Metric file to set properties to
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_display properties_ (such as `y-label`, `template`, `title`, etc.) each time
+plots are generated with `dvc plot show` or `dvc plot diff`. This command sets
+(or unsets) default display properties for a specific metrics file.
+
+The path to the metrics file `target` is required. It must be listed in a
+[`dvc.yaml`](/doc/user-guide/dvc-file-format) file (see the `--plots` option of
+`dvc run`). `dvc plots modify` adds the display properties to `dvc.yaml`.
+
+The available display properties are expressed as the [options](#options) of
+this command (prefixed with `--`).
+
+> Display properties are based on the
```

Comment:
let's un-note it and make a part of the previous paragraph? `The full list of properties ....`

Reply:
No strong opinion about removing block quote but it is kind of an external reference and from what I understand it wouldn't be correct to say "the full list of props" because we don't provide a way to change all of them, only the specific options of this command. Can you please confirm, @pared ? Thanks

Reply:
The way I see it - it explains where can I get and see available option names right (among other things, like a reference to the Vega, etc). It feels like it's very important info. It should be part of the previous paragraph where we explain how to pass them.

We indeed should clarify re the all of some of the options availability - ask directly in the eng channel?

Reply:
> It feels like it's very important info. It should be part of the previous paragraph

I did move this into the prev p. and remove block quote. But I'm not sure that this is super important because plots modify doesn't let you change all those props. We do link to this page from a few other places, including from the plots index main description. I'll double check with eng and extract to #1414.